pkg/dns/vips/interfaces.go

```diff
@@ -7,3 +7,11 @@ func (vips List) Append(other List) {
 		vips[k] = v
 	}
 }
+
+func (vips List) DomainsByIPs() map[string]string {
```

Comment:
Should that be `FQDNsByIPs`?
https://en.wikipedia.org/wiki/Fully_qualified_domain_name